### PR TITLE
certbot deps: python-time-machine, python-freezegun, python-distlib, python-pytest-env

### PR DESCRIPTION
**Pull request by a16bitsysop**

### CODE REVIEW

File: contrib/python-pytest-env/template.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-hatchling",
+    "python-hatch_vcs",
+    "python-wheel",
+]
+depends = ["python-pytest"]
+checkdepends = depends
```
Comment (nekopsykose) [inline contrib/python-pytest-env/template.py:13]:
```suggestion
checkdepends = list(depends)
```

File: contrib/python-py-cpuinfo/template.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+pkgname = "python-py-cpuinfo"
```
Comment (nekopsykose) [inline contrib/python-py-cpuinfo/template.py:1]: added this for benchmark so you can rebase it out

File: contrib/python-distlib/template.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = ["python-pytest"]
+pkgdesc = (
+    "Low-level functions for packaging and distribution of Python software"
+)
```
Comment (nekopsykose) [inline contrib/python-distlib/template.py:18]:
```suggestion
pkgdesc = "Functions for packaging and distribution of Python software"
```

File: contrib/python-pytest-env/template.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+pkgname = "python-pytest-env"
+pkgver = "1.1.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-hatchling",
+    "python-hatch_vcs",
+    "python-wheel",
```
Comment (nekopsykose) [inline contrib/python-pytest-env/template.py:10]: you can leave wheel out when hatchling is used

File: contrib/python-py-cpuinfo/template.py (hunk 2 above)
Comment (a16bitsysop) [reply contrib/python-py-cpuinfo/template.py:1]: Okay will drop

File: contrib/python-pytest-env/template.py (hunk 4 above)
Comment (a16bitsysop) [reply contrib/python-pytest-env/template.py:10]: Okay will take out

**Pull request merged**